tests/unit/common/util/stringify.test.js

```diff
@@ -19,10 +19,12 @@ test('should utilize JSON.stringify to serialize input data', () => {
 test.each([
   ['array', [0, 1, 'asdf', undefined, null, 'weee'], '[0,1,"asdf",null,null,"weee"]'],
   ['object', { a: 123, c: 'b', f: 'asdf', u: undefined, n: null }, '{"a":123,"c":"b","f":"asdf","n":null}'],
-  ['undefined', undefined, undefined],
+  ['undefined', undefined, ''],
   ['null', null, 'null'],
   ['number', 123, '123'],
-  ['function', function F () {}, undefined],
+  ['string', '0', '"0"'],
+  ['symbol', Symbol('foo'), ''],
+  ['function', function F () {}, ''],
```

Comment:
Assuming we are to follow the native `JSON.stringify` behavior for functions, which returns `undefined`, which in turn becomes `''` in our util method (as opposed to, the function `toString()` method which returns the entire function as a string).

Reply:
For brevity, can you also add a case to this unit test that implements the toJSON attribute on an object?

Reply:
```suggestion
  ['function', function F () {}, ''],
  ['toJSON valid', { toJSON: () => ({foo:'bar'})}, '{"foo":"bar"}'],
  ['toJSON undefined', { toJSON: () => undefined}, ''],
  ['toJSON error', { toJSON: () => {throw new Error('toJSON')}}, ''],
```